src/System.Net.Sockets/src/System/Net/Sockets/UdpReceiveResult.cs

```diff
@@ -4,12 +4,19 @@
 
 namespace System.Net.Sockets
 {
-    public struct UdpReceiveResult : IEquatable<UdpReceiveResult>
-    {
+	/// <summary>
+	/// Presents UDP receive result information from a call to the <see cref="UdpClient.ReceiveAsync"/> method
+	/// </summary>
+	public struct UdpReceiveResult : IEquatable<UdpReceiveResult> {
```

Comment:
According to the CoreFX [coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md), Allman style braces should be used, where each brace begins on a new line.

Reply:
Auto changed by VSSettings. Changed

Reply:
The brace still isn't on a new line.

Reply:
done